core/bourbon/helpers/_contrasting-color.scss

```diff
@@ -0,0 +1,7 @@
+@function contrasting-color($background-color, $light-color: #000, $dark-color: #fff) {
```

Comment:
It might not always be text, though…I could use this function to contrast two shapes with color. I see what you mean with `background-color`, but I chose that because it would be the color _behind_ whatever you’re trying to contrast with. Maybe it should just be `$base-color`?